loleaflet/src/map/handler/Map.FileInserter.js

```diff
@@ -85,6 +85,11 @@ L.Map.FileInserter = L.Handler.extend({
 			this._sendURL(name, this._toInsertURL[name]);
 		}
 		this._toInsertURL = {};
+
+		for (name in this._toInsertBackground) {
```

Comment:
Would be good to comment what's going on here. 

It's not clear at all why we send the background file from the `_onChildIdMsg` handler, and it's clear that there is some non-obvious workflow where we set the _toInsertBackground array separately and consume it here. Best to document what's going on somewhere, and put short comments at the different stages so that maintainers can follow through when they need to.